src/components/Home.js

```diff
@@ -1,23 +1,32 @@
 import * as React from 'react';
+import { useFilter } from '../contexts/filter-context';
 import { useNote } from '../contexts/note-context';
 import { Navbar, SearchBar, EmptyNote, Note } from './customComponents';
 
 export const Home = () => {
-    const {notes} = useNote();
+    const {notes, saveNote } = useNote();
+    const {filteredState} = useFilter();
+    filteredState && console.log(`filteredNotes in Home: ${JSON.stringify(filteredState.filteredNotes)}`)
+    const allNotes = notes && notes.filter(note => note.state !== 'DELETED');
+    const displayNotes = filteredState ? filteredState.filteredNotes.length===0 ? allNotes : filteredState.filteredNotes : allNotes;
+    console.log(`displayNotes in Home: ${JSON.stringify(displayNotes)}`)
```

Comment:
Apply in same

Reply:
Will remove them after dev complete. Need them for debugging purpose.